Bellerophon/Bellerophon/Sources/BPEvent.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  BPTypes.swift
+//  Bellerophon
+//
+//  Created by Satinder Singh on 11/17/17.
+//
+
+/// Different types of events for Bellerophon
+@objc
+public enum BellerophonEvent: Int {
```

Comment:
suggestion, perhaps an enum with associated value. So 
```
public enum BellerophonEvent: Int {
  case killSwitch(view: UIView)
  case forceUpdate(view: UIView)
}
```

not sure how compatible this will be with Objc though

Reply:
This suggestion is not compatible with Objective-C. So I'll need to continue making significant changes by now removing `@objc` from the codebase essentially

Reply:
Suspected so, if not compatible with objc, I'd come up with a different approach. It's definitely do-able while maintaining backwards compatibility.